tekton-pipelines/base/tasks/slack-notification-task.yaml

```diff
@@ -28,7 +31,10 @@ spec:
       - configMapRef:
           name: $(params.application)-build-pipeline-config
       - secretRef:
-          name: $(params.application)-slack-secret  # used for webhook URL
+          name: $(params.application)-slack-secret  # used for webhook URL, channel_id
+      - secretRef:
+          name: slack-api-token-secret  # used for slack api token (common for all projects)
```

Comment:
include info how to obtain that token secret. Keep in mind our repo is open to the world. so it should be instructions, now Link to slack UI